packages/manager/modules/storage/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@ovh-ux/manager-storage",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud storage product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/storages"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.15"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^9.0.0",
```

Comment:
cloud universe components is used but is missing in peer deps

Reply:
Package `@ovh-ux/ng-ovh-cloud-universe-components` has been added as a peer dependency.